gammapy/makers/safe.py

```diff
@@ -66,6 +70,7 @@ def __init__(
         self.aeff_percent = aeff_percent
         self.bias_percent = bias_percent
         self.position = position
+        self.fixed_offset = fixed_offset
```

Comment:
I think `position` and `fixed_offset` are mutually exclusive, so maybe add a check here in `__init__`

Reply:
Ok. I've added a condition to check that the two arguments are not passed together.